File: 1° Bimestre/Aula 0221/Media.java
import javax.swing.JOptionPane;

public class Media {
    public static void main(String[] args){
        String nome;
        double media=0;
        Double [] notas = new Double[4];

        nome = JOptionPane.showInputDialog("Digite o nome do aluno:");

        for(int i=0;i<notas.length;i++){
            notas[i]=Double.parseDouble(JOptionPane.showInputDialog("Digite a nota do "+(i+1)+"° bimestre."));
            media+=notas[i];
        }

        media = media/4;
        System.out.println("Nome do aluno: "+nome);

        System.out.println("Notas:");
        for(int i=0;i<notas.length;i++){
            System.out.println((i+1) +"° Bimestre: " +notas[i]);
        }
        System.out.println("Média: "+media);
    }
}

File: 1° Bimestre/Aula 0314/ex3/media.php
<!DOCTYPE html>
<html lang="pt-br">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Média Aluno</title>
    <style>
        body{
            font-family: Arial, Helvetica, sans-serif;
            text-align: center;
            font-weight: bold;
        }

        .aprovado{
            color: green;
        }

        .reporvado{
            color: red;
        }
    </style>
</head>
<body>
    <?php
        $nome = $_POST['nome'];
        $nota1 = $_POST['nota1'];
        $nota2 = $_POST['nota2'];
        $nota3 = $_POST['nota3'];
        $nota4 = $_POST['nota4'];
        
        if($nota1 == "" || $nota2 == "" || $nota3 == "" || $nota4 == ""){
            echo "Você não digitou todas as notas.";
        }else{
            if($nome != ""){
                if($nota1 > 0 && $nota2 > 0 && $nota3 > 0 && $nota4 > 0 && $nota1 <= 10&& $nota2 <= 10&& $nota3 <= 10&& $nota4 <= 10){
                    $media = ($nota1 + $nota2 + $nota3 + $nota4)/4;
                    if($media >= 5){
                        echo "<div class=aprovado>O(a) aluno(a) $nome foi aprovado</div>Média: $media";
                    }else{
                        echo "<div class=reporvado>O(a) aluno(a) $nome foi reprovado</div> Média: $media";
                    }
                }else{
                    echo "As notas variam de 1 a 10, digite valores válidos.";
                }
            }else{
                echo "Digite o nome do aluno.";
            }
        }
    ?>
</body>
</html>